ui/components/app/assets/defi-list/cells/defi-empty-state.tsx

```diff
@@ -1,45 +1,62 @@
-import React, { FC } from 'react';
-import { Box, Text } from '../../../../component-library';
+import React, { FC, useCallback, useContext } from 'react';
+import { useSelector } from 'react-redux';
+import { useI18nContext } from '../../../../../hooks/useI18nContext';
+import { useTheme } from '../../../../../hooks/useTheme';
+import { TabEmptyState } from '../../../../ui/tab-empty-state';
+import { ThemeType } from '../../../../../../shared/constants/preferences';
+import { getPortfolioUrl } from '../../../../../helpers/utils/portfolio';
+import { MetaMetricsContext } from '../../../../../contexts/metametrics';
 import {
-  Display,
-  JustifyContent,
-  AlignItems,
-  FlexDirection,
-  TextColor,
-  TextVariant,
-  TextAlign,
-} from '../../../../../helpers/constants/design-system';
+  MetaMetricsEventCategory,
+  MetaMetricsEventName,
+} from '../../../../../../shared/constants/metametrics';
+import {
+  getDataCollectionForMarketing,
+  getMetaMetricsId,
+  getParticipateInMetaMetrics,
+} from '../../../../../selectors';
+
+export const DeFiEmptyStateMessage: FC = () => {
+  const t = useI18nContext();
+  const theme = useTheme();
+  const trackEvent = useContext(MetaMetricsContext);
+
+  const metaMetricsId = useSelector(getMetaMetricsId);
+  const isMetaMetricsEnabled = useSelector(getParticipateInMetaMetrics);
+  const isMarketingEnabled = useSelector(getDataCollectionForMarketing);
+
+  const handleExploreDefi = useCallback(() => {
+    const url = getPortfolioUrl(
+      'explore/tokens',
+      'ext_defi_empty_state_button',
+      metaMetricsId,
+      isMetaMetricsEnabled,
+      isMarketingEnabled,
+    );
+    global.platform.openTab({ url });
+    trackEvent({
+      category: MetaMetricsEventCategory.Navigation,
+      event: MetaMetricsEventName.PortfolioLinkClicked,
+      properties: {
+        location: 'DeFiTab',
+        text: 'Explore DeFi',
+      },
+    });
+  }, [isMarketingEnabled, isMetaMetricsEnabled, metaMetricsId, trackEvent]);
+
+  const defiIcon =
+    theme === ThemeType.dark
+      ? '/images/empty-state-defi-dark.png'
+      : '/images/empty-state-defi-light.png';
 
-export const DeFiEmptyStateMessage: FC<{
-  primaryText: string;
-  secondaryText: string;
-}> = ({ primaryText, secondaryText }) => {
   return (
-    <Box
-      paddingTop={6}
-      marginTop={12}
-      marginBottom={12}
-      display={Display.Flex}
-      justifyContent={JustifyContent.center}
-      alignItems={AlignItems.center}
-      flexDirection={FlexDirection.Column}
-      className="nfts-tab__link"
```

Comment:
Checked for this classname and think it was used incorrectly here. 

<img width="463" height="522" alt="Screenshot 2025-09-18 at 5 17 10 PM" src="https://github.com/user-attachments/assets/dbd7c25e-66e2-40c2-a34b-1ad50765f7b8" />